Week3/homework/step4.js

```diff
@@ -1,7 +1,17 @@
 'use strict';
 
-// Add your code here
+function createBase(num) {
+  return function (item) {
+    return num + item;
+  }
+}
 
+const addSix = createBase(6);
+addSix(10); // returns 16
+addSix(21); // returns 27
+
+/* Second way to structuring the Function*/
+createBase = base => console.log(base + num);
```

Comment:
See my comment above about reusing names.

This second version does something different (it use console.log) than the first version.